### PR TITLE
Feature/authorization frontend

### PR DESCRIPTION
**Pull request by VL4DYSL4V**

### CODE REVIEW

File: slidez-fe-core/src/App.tsx [hunk 1]
```diff
@@ -14,4 +18,21 @@ function App() {
   )
 }
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/App.tsx:19]: I think, you can drop it

File: slidez-fe-core/src/App.tsx [hunk 2]
```diff
@@ -14,4 +18,21 @@ function App() {
...
+            exact
+            path={[AppRoute.LOGIN, AppRoute.REGISTRATION]}
+            component={SignPage}
+          />
+          <Route path={AppRoute.ANY} exact component={DefaultApp} />
+        </Switch>
+      </HashRouter>
+    </div>
+  )
+}
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/App.tsx:36]: And I'd rather move routing to a separate component, as here may be some more data

File: slidez-fe-core/src/common/components/forms/LoginForm.tsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+              icon={faEye}
+              className='input-icon'
+              onClick={onRevealClick}
+            />
+          ) : (
+            <FontAwesomeIcon
+              icon={faEyeSlash}
+              className='input-icon'
+              onClick={onRevealClick}
+            />
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/common/components/forms/LoginForm.tsx:101]: Will it work if you set icon = {isPasswordRevealed ? faEye : faEyeSlash}?

File: slidez-fe-core/src/common/components/forms/sign-form.css [hunk 4]
```diff
@@ -0,0 +1,158 @@
+div {
+    --grey: #999999;
+    --mint: #4af092;
+    --light-grey: #eeeeee;
+    --white: white;
+    --black: #222222;
+    --error: #f73b48;
+    --error-background: #ffb8bc;
+}
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/common/components/forms/sign-form.css:9]: Move variables to some global css file
And don't we use scss or sass?

File: slidez-fe-core/src/containers/counter/Counter.tsx [hunk 5]
```diff
@@ -51,13 +51,13 @@ export function Counter() {
           Add Amount
         </button>
         <button
-          className={styles.asyncButton}
+          className={styles.asyncButton} // @ts-ignore
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/containers/counter/Counter.tsx:54]: What was wrong with it?

And you can delete this file, then

File: slidez-fe-core/src/App.tsx (hunk 1 above)
Comment (VladislavArzamastsev) [reply slidez-fe-core/src/App.tsx:19]: Am I getting this correct: I need to change my css to scss, refactor weird icon thing, move variables to global css file and delete stuff that was created by "create-react-appication" command?

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by VolodymyrYavorskyi — APPROVED** [review]

**Pull request merged**